src/test/java/bio/terra/service/filedata/azure/util/BlobContainerCopyInfoTest.java

```diff
@@ -1,32 +1,30 @@
 package bio.terra.service.filedata.azure.util;
 
-import static com.azure.core.util.polling.LongRunningOperationStatus.*;
+import static com.azure.core.util.polling.LongRunningOperationStatus.FAILED;
+import static com.azure.core.util.polling.LongRunningOperationStatus.IN_PROGRESS;
+import static com.azure.core.util.polling.LongRunningOperationStatus.SUCCESSFULLY_COMPLETED;
+import static com.azure.core.util.polling.LongRunningOperationStatus.USER_CANCELLED;
 import static com.azure.storage.blob.models.CopyStatusType.PENDING;
 import static com.azure.storage.blob.models.CopyStatusType.SUCCESS;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 import com.azure.core.util.polling.LongRunningOperationStatus;
 import com.azure.core.util.polling.PollResponse;
-import com.azure.core.util.polling.SyncPoller;
 import com.azure.storage.blob.models.BlobCopyInfo;
 import com.azure.storage.blob.models.CopyStatusType;
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
 
-@ExtendWith(MockitoExtension.class)
+@Tag("bio.terra.common.category.Unit")
```

Comment:
This test was written using JUnit 5 parameterized test APIs but had never been added to a category so it was never run.